### PR TITLE
Update docs according to v10.0.1 release

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
can you also include a deprecation warning for `avm.getTxStatus` similar to the one in this picture? applies for v 1.10.0 

see here for explanation: https://docs.avax.network/apis/avalanchego/x-chain-migration#transaction-broadcast-and-confirmation

image

File: docs/apis/avalanchego/apis/p-chain.md [hunk 1]
```diff
@@ -1590,6 +1591,7 @@ platform.getStake({
 ```
 
 - `addresses` are the addresses to get information about.
+- `validatorsOnly`
```
Comment (meaghanfitzgerald) [inline docs/apis/avalanchego/apis/p-chain.md:1594]: please add a description here

**Review by meaghanfitzgerald — CHANGES_REQUESTED** [review]
Only one comment otherwise LGTM

File: docs/apis/avalanchego/apis/p-chain.md (hunk 1 above)
Comment (laviniat1996) [reply docs/apis/avalanchego/apis/p-chain.md:1594]: Funny thing, but I have added a description. Weird that it doesn't show up.

Comment (laviniat1996) [reply docs/apis/avalanchego/apis/p-chain.md:1594]: Turns out I didn't save it before pushing the commit lol

Comment (laviniat1996) [reply docs/apis/avalanchego/apis/p-chain.md]: f68303b6c7a98d2ef46dd531688988f5b2d26bae should show up now

File: docs/apis/avalanchego/apis/x-chain.md [hunk 2]
```diff
@@ -1144,6 +1149,7 @@ An output contains:
   `FvwEAhmxKfeiG8SnEvq42hc6whRyY3EFYAvebMqDNDGCgxN5Z`.)
 - `fxID`: The ID of the FX this output uses.
 - `output`: The FX-specific contents of this output.
+- `id`: The transaction ID
```
Comment (StephenButtolph) [inline docs/apis/avalanchego/apis/x-chain.md:1152]: An output does not include a transaction ID

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 3]
```diff
@@ -955,6 +955,10 @@ Frequency of renewing this node's average uptime metric. Defaults to `30s`.
 
 ##### `--snow-avalanche-batch-size` (int)
 
+:::caution
```
Comment (StephenButtolph) [inline docs/nodes/maintain/avalanchego-config-flags.md:958]: This was not deprecated - it was fully removed

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 4]
```diff
@@ -963,6 +967,10 @@ batch, and will then issue the entire batch at once. The value must be at least
 
 ##### `--snow-avalanche-num-parents` (int)
 
+:::caution
```
Comment (StephenButtolph) [inline docs/nodes/maintain/avalanchego-config-flags.md:970]: This isn't deprecated - it was deleted

File: docs/apis/avalanchego/apis/x-chain.md (hunk 2 above)
Comment (laviniat1996) [reply docs/apis/avalanchego/apis/x-chain.md]: fixed 1fa6ed428ac455d0293f6cd97289fd50fb6764e2

File: docs/nodes/maintain/avalanchego-config-flags.md (hunk 3 above)
Comment (laviniat1996) [reply docs/nodes/maintain/avalanchego-config-flags.md]: true, my bad. fixed 1fa6ed428ac455d0293f6cd97289fd50fb6764e2

File: docs/nodes/maintain/avalanchego-config-flags.md (hunk 4 above)
Comment (laviniat1996) [reply docs/nodes/maintain/avalanchego-config-flags.md]: fixed 1fa6ed428ac455d0293f6cd97289fd50fb6764e2

**Pull request merged**